internal/pkg/v2/infrastructure/test/acceptance.go

```diff
@@ -0,0 +1,80 @@
+/*******************************************************************************
+ * Copyright 2020 Dell Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ *******************************************************************************/
+
+package test
+
+import (
+	"net/http"
+	"testing"
+
+	"github.com/gorilla/mux"
+	"github.com/stretchr/testify/assert"
+)
+
+// UseCaseAcceptance implements the common acceptance test for use-cases via use-case endpoint.
+func UseCaseAcceptance(
+	t *testing.T,
+	router *mux.Router,
+	endpoint string,
+	method string,
+	versionVariations map[string][]*Case) {
+
+	for version := range versionVariations {
```

Comment:
This function could use some additional comments...

Reply:
I'm not sure what's not clear here so I'm not sure what additional comments are required.